### PR TITLE
RF 00.01 | Restore resources.TEMPLATE

### PR DESCRIPTION
**Pull request by Albina-star**
https://trello.com/c/ALm3BXUD/933-rf-0001-restore-resourcesteplate

### CODE REVIEW

File: src/test/resources/local.properties.TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,4 @@
+local.admin.username=<>
+local.admin.password=<>
+local.port=8080
+local.chrome_options=--window-size=1920,1080
```
Comment (SergeiKuznetsov) [inline src/test/resources/local.properties.TEMPLATE:4]:
```suggestion
local.chrome_options=--window-size=1920,1080

```

Comment (Albina-star) [reply src/test/resources/local.properties.TEMPLATE]: ok

Comment (SergeiKuznetsov) [reply src/test/resources/local.properties.TEMPLATE]: А две строки зачем? ))

**Comment by SergeiKuznetsov** [comment]
И опечатка в названии файла src/test/resources/local.properties.TEMPLATE

File: src/test/resources/local.properties.TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (SergeiKuznetsov) [inline src/test/resources/local.properties.TEMPLATE:5]:
```suggestion
```

**Review by SergeiKuznetsov — APPROVED** [review]

**Comment by SergeiKuznetsov** [comment]
Какие-то тесты упали
`Results :

Failed tests:   testMoveMultibranchPipelineToFolder(school.redrover.MultibranchPipeline3Test): no such element: Unable to locate element: {"method":"xpath","selector":"//button[@name='Submit']"}(..)
  testCreateMultiConfigurationProjectInFolder(school.redrover.CreateMultiConfigurationProjectTest): no such element: Unable to locate element: {"method":"css selector","selector":"*[name='Submit']"}(..)

Tests run: 282, Failures: 2, Errors: 0, Skipped: 0`

**Review by SergeiKuznetsov — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]

**Comment by SergeiKuznetsov** [comment]
```
Results :

Failed tests:   testClickLogoToReturnToDashboardPage(school.redrover.PageHeaderTest): no such element: Unable to locate element: {"method":"xpath","selector":"//button[normalize-space()='Save']"}(..)
  testCreateOrganizationFolder(school.redrover.MultibranchPipelineTest): no such element: Unable to locate element: {"method":"xpath","selector":"//div[@id='view-message']"}(..)

Tests run: 282, Failures: 2, Errors: 0, Skipped: 0
```

**Pull request merged**